subsys/net/lib/mqtt/Kconfig

```diff
@@ -35,4 +35,17 @@ config MQTT_LIB_WEBSOCKET
 	help
 	  Enable Websocket support for socket MQTT Library.
 
+config MQTT_CLEAN_SESSION
+	int "MQTT Clean Session Flag."
```

Comment:
If there is only two possible values 0/1, we should make this a boolean as it would make it more intuitive and user would not be able to feed any value.

Reply:
I made it bool but then the change in `mqtt_internal.h` looks like this, is it ok?
```
/**@brief Clean session on every connect (1) or keep subscriptions and messages
 *        between connects (0)
 */
#ifdef CONFIG_MQTT_CLEAN_SESSION
#define MQTT_CLEAN_SESSION 1U
#else
#define MQTT_CLEAN_SESSION 0U
#endif
```